### PR TITLE
[#160613728] Configure error handler with Instabug crash reporting.

### PR DESCRIPTION
**Pull request by DenisFrezzato**
I added two CircleCi jobs (one for each platform) which generate the sourcemap and upload them to Instabug. The CI environment must have the `INSTABUG_TOKEN` environment variable in order to work.

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#160613728](https://www.pivotaltracker.com/story/show/160613728): Come product owner voglio poter raccogliere informazioni su crash dell'app tramite Instabug

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: .circleci/config.yml [hunk 1]
```diff
@@ -209,6 +209,27 @@ jobs:
...
+          name: Build sourcemap
+          command: yarn bundle:android-sourcemap
+      
+      - run:
+          name: Install jq
+          command: apt-get install -y jq
+
+      - run:
+          name: Build sourcemap and upload it to Instabug
+          command: ./scripts/instabug-upload-sourcemap.sh android
```
Comment (cloudify) [inline .circleci/config.yml]: how is the sourcemap linked to the commit number it has been generated from?

**Comment by cloudify** [comment]
Thanks @DenisFrezzato , I think we also need to remove https://github.com/master-atul/react-native-exception-handler

File: .circleci/config.yml (hunk 1 above)
Comment (DenisFrezzato) [reply .circleci/config.yml]: In the documentation there is nothing about that, so I don't actually know. I suppose Instabug get the app version from both the error reporting and the sourcemap.

**Comment by DenisFrezzato** [comment]
@cloudify I reverted the removal of `react-native-exception-handler`.

**Comment by cloudify** [comment]
@DenisFrezzato do I need to set the INSTABUG_TOKEN env var on circleci?

**Comment by DenisFrezzato** [comment]
@cloudify Yes! Is necessary to upload the sourcemaps.

File: ts/react-native-source-maps/index.ts [hunk 2]
```diff
@@ -1,88 +0,0 @@
...
-  - `createSourceMapper` method was customized in order to add support for Android
-  we could either fork or merge back once we're positive about our solution
-*/
-
-// Temporary disable eslint for this file
-/* eslint-disable */
-
-import { Platform } from "react-native";
-import RNFS from "react-native-fs";
-import SourceMap, { MappedPosition } from "source-map";
```
[the remark this answers is not in the document]
Comment (DenisFrezzato) [reply ts/react-native-source-maps/index.ts]: I removed both `source-map` and `react-native-fs`.

**Comment by cloudify** [comment]
@DenisFrezzato can you also add the instabug token to the circleci conf? it's ok since it's already public

**Pull request closed without merging**